x/group/server/msg_server.go

```diff
@@ -604,6 +646,43 @@ type authNGroupReq interface {
 }
 
 type actionFn func(m *group.GroupInfo) error
+type groupAccountActionFn func(m *group.GroupAccountInfo) error
+
+// doUpdateGroupAccount first makes sure that the group admin initiated the group account update,
+// before performing the group account update and emitting an event.
+func (s serverImpl) doUpdateGroupAccount(ctx types.Context, groupAccount string, admin string, action groupAccountActionFn, note string) error {
+	groupAccountAddress, err := sdk.AccAddressFromBech32(groupAccount)
+	if err != nil {
+		return sdkerrors.Wrap(err, "group admin")
+	}
+
+	var groupAccountInfo group.GroupAccountInfo
+	err = s.groupAccountTable.GetOne(ctx, groupAccountAddress.Bytes(), &groupAccountInfo)
+	if err != nil {
+		return sdkerrors.Wrap(err, "load group account")
+	}
+
+	groupAdmin, err := sdk.AccAddressFromBech32(admin)
```

Comment:
Just renaming this for the sake of clarity:
```suggestion
	groupAccountAdmin, err := sdk.AccAddressFromBech32(admin)
```